src/modules/components/asset/assetTransfer/assetTransfer.tsx

```diff
@@ -0,0 +1,187 @@
+
+    const resolvedSenderLink =
+        blockExplorerUrl && senderAddress ? `${blockExplorerUrl}/address/${senderAddress}` : undefined;
+    const resolvedRecipientLink =
+        blockExplorerUrl && recipientAddress ? `${blockExplorerUrl}/address/${recipientAddress}` : undefined;
+
+    const formattedTokenValue = formatterUtils.formatNumber(tokenAmount && tokenAmount > 0 ? tokenAmount : null, {
+        format: NumberFormat.TOKEN_AMOUNT_SHORT,
+        withSign: true,
+    });
+    const fiatValue = Number(tokenAmount ?? 0) * Number(tokenPrice ?? 0);
+    const formattedFiatValue = formatterUtils.formatNumber(fiatValue, {
+        format: NumberFormat.FIAT_TOTAL_SHORT,
+    });
+    const formattedTokenAmount = formattedTokenValue && tokenSymbol ? `${formattedTokenValue} ${tokenSymbol}` : `-`;
+
+    return (
+        <div className="flex h-full w-[320px] flex-col gap-y-2 md:w-[640px] md:gap-y-3">
+            <div className="relative flex h-full  flex-col rounded-xl border-[1px] border-neutral-100 md:flex-row">
+                <a
+                    href={resolvedSenderLink}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className={classNames(
+                        'flex h-20 w-full items-center space-x-4 rounded-l-xl px-4 py-7', //base
+                        'hover:border-neutral-200 hover:shadow-neutral-md', //hover
+                        'focus:outline-none focus-visible:rounded-l-xl focus-visible:ring focus-visible:ring-primary focus-visible:ring-offset', //focus
+                        'active:border-[1px] active:border-neutral-300', //active
+                        'md:w-1/2 md:p-6', //responsive
+                    )}
+                >
+                    <MemberAvatar responsiveSize={{ md: 'md' }} ensName={senderEnsName} address={senderAddress} />
+                    <div className="flex flex-col">
+                        <p className="text-xs font-normal leading-tight text-neutral-500 md:text-sm">From</p>
+                        <div className="flex items-center space-x-1">
+                            <p className="text-sm font-normal leading-tight text-neutral-800 md:text-base">
+                                {resolvedSenderHandle}
+                            </p>
+                            <Icon icon={IconType.LINK_EXTERNAL} size="sm" className="text-neutral-300" />
+                        </div>
+                    </div>
+                </a>
+                <div className="border-t-[1px] border-neutral-100 md:border-l-[1px]" />
+                <AvatarIcon
+                    icon={IconType.CHEVRON_DOWN}
+                    size="sm"
+                    className={classNames(
+                        'absolute left-4 top-1/2 -translate-y-1/2 bg-neutral-50 text-neutral-300', //base
+                        'md:left-1/2 md:-translate-x-1/2 md:-rotate-90', //responsive
+                    )}
+                />
```

Comment:
So much code that could be reused, please consider implementing an internal `AssetTransferAddress` component to render the sender/recipient of the transfer